app/views/repositories/show.html.erb

```diff
@@ -59,7 +59,10 @@
              }
   %>
 
-  <div id="actionToolbar" data-behaviour="vue">
+  <div
+    id="actionToolbar" 
+    data-behaviour="vue" 
```

Comment:
why did you format like this?

Reply:
Just forgot to remove after reverting previous changes.